Resources/views/custom.css.twig

```diff
@@ -270,7 +281,16 @@ abbr[title]  {
     padding-right: 30px;
 }
 .pumukit_mmobj {
-    border-radius: 0;
+    border-radius: 2;
```

Comment:
Without adding an unit ('px' for example), the border-radius property doesn't work and defaults to 10px